src/toil/resource.py

```diff
@@ -335,8 +328,8 @@ def _load(cls, path):
 
 
 class ModuleDescriptor(namedtuple('ModuleDescriptor', ('dirPath', 'name', 'fromVirtualEnv'))):
-    """
-    A path to a Python module decomposed into a namedtuple of three elements, namely
+    r"""
+    A path to a Python module decomposed into a namedtuple of three elements/
```

Comment:
Why the slash (and `r` string designation I assume to accommodate it)?

Reply:
No good reason! I'll fix it